terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java

```diff
@@ -323,7 +323,8 @@ public void setColumns(Map<String, Map<Locale, String>> cols) {
 
     /**
      * Sets the default locale as fallback.<br>
-     * Defaults to {@link Locale#getDefault()}
+     * If extend this and override default value of {@link SimpleI18nCodeList#fallbackTo fallbackTo}, affects
+     * {@link SimpleI18nCodeList#afterPropertiesSet afterPropertiesSet}.
```

Comment:
- [ ] You can omit own class name of `@link`.

`{@link SimpleI18nCodeList#afterPropertiesSet afterPropertiesSet}`
-> `{@link #afterPropertiesSet afterPropertiesSet}`

Reply:
Modified by ead480234bf63b203affa7ba320af662494a397b

Reply:
You must correct `{@link java.util.Locale#getDefaul Locale#getDefault}` to `{@link Locale#getDefault Locale#getDefault}`.

Reply:
Modified by 36fda44
I forgot to paste the link.
All fixes of `java.util` are carried out collectively. (https://github.com/terasolunaorg/terasoluna-gfw/pull/874#discussion_r262028454)